uint32 pps_counter
uint32 sync_sec
uint32 sync_nsec